src/main/java/piece/Byeong.java

```diff
@@ -0,0 +1,52 @@
+package piece;
+
+import static coordinate.Direction.DOWN;
+import static coordinate.Direction.LEFT;
+import static coordinate.Direction.RIGHT;
+
+import board.Board;
+import coordinate.Coordinate;
+import coordinate.MoveVector;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import team.Team;
+
+public class Byeong extends Piece {
+
+    public Byeong(Team team) {
+        super(team);
+        if (team == Team.CHO) {
+            throw new IllegalArgumentException("병은 초나라에서 사용할 수 없습니다.");
+        }
+    }
+
+    @Override
+    protected Set<Coordinate> findMovableCandidates(Coordinate departure) {
+        return Stream.<List<MoveVector>>of(
+                        List.of(DOWN),
+                        List.of(RIGHT),
+                        List.of(LEFT)
```

Comment:
특별한 이유가 없다면 리스트를 1개만 활용해볼까요?

```suggestion
                        List.of(DOWN, RIGHT, LEFT)
```

Reply:
이것도 오버로딩할 수 있도록 로직을 두 개 만들어뒀습니다!